_posts/2021-07-08-how-to-upgrade-from-opendistro-to-opensearch.markdown

```diff
@@ -0,0 +1,250 @@
+---
+layout: post
+title:  How To: Upgrade from Open Distro to OpenSearch
```

Comment:
The colon is causing a YAML parsing error 🤦‍♂️

Can you enclose the title in double quotes?

Reply:
Ah sure that make sense.